dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -86,7 +95,16 @@ _wait_for_cluster_ready(){
             echo "Set end time of prom scrape"
             export END_TIME=$(date +"%s")       
             START_TIMER=$(date +%s)
-            _wait_for_nodes_ready $1 ${COMPUTE_WORKERS_NUMBER} "node-role.kubernetes.io/worker"
+            if [ "$ENABLE_SPOT_WORKERS" == "true" ]; then
+                if [ "$SPOT_POOL_READY" == "true" ]; then
+                    _wait_for_nodes_ready $1 ${COMPUTE_WORKERS_NUMBER} "node-role.kubernetes.io/worker"
+                else
+                    DEFAULT_WORKER_NODES=3
```

Comment:
Shouldn't the default be always `openshift_worker_count`?

Reply:
openshift_worker_count=9
The workflow will be, 
default cluster= 3(M+W-I)
We already added 3 nodes, the remaining 6 nodes would be SPOT nodes.
So I added the DEFAULT_WORKER_NODES=3 to verify that 3 nodes in ready state.